stockflux-launcher/README.md

```diff
@@ -1 +1,41 @@
-# StockFlux Launcher
+## StockFlux Launcher
+
+StockFlux Launcher is a part of StockFlux app suite intended to work with StockFlux Watchlist, Chart, and News apps. The launcher is considered to be the centre of the StockFlux application suite. It is used to launch the other applications either directly via icons or by searching for a symbol and opening using the results returned.
+
+### Running locally
+
+In order to launch the StockFlux Launcher app locally, StockFlux Core and StockFlux Components have to be built first.
+
+```bash
+cd stockflux-core
+npm run build
+cd stockflux-components
+npm run build
+```
+
+Then install StockFlux Launcher dependencies.
+
+```bash
+cd stockflux-launcher
+npm install
+```
+
+To start and launch the StockFlux Launcher:
+
+```bash
+npm start
+npm run launch
+```
+
+To launch other StockFlux apps locally read the `README.md` files in the project folders.
+
+### Launching StockFlux Launcher app from the installer
```

Comment:
I'd say can we re-jig all of these so the standalone documentation (i.e. how to run it) comes before the developer/contributors notes?